third-party/spamassassin/pom.xml

```diff
@@ -84,6 +88,17 @@
             <artifactId>james-server-data-memory</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-guice-common</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-guice-configuration</artifactId>
+            <version>${james.baseVersion}</version>
```

Comment:
Is this really necessary? Logically you should not have to redeclare versions here? Idem for other concerned deps in this pom

Reply:
I copied from the `examples` module. But yes, It's not necessary